Tribler/Test/Core/test_torrent_def.py

```diff
@@ -28,6 +35,13 @@ def __init__(self, *argv, **kwargs):
         super(TestTorrentDef, self).__init__(*argv, **kwargs)
         self._logger = logging.getLogger(self.__class__.__name__)
 
+    def setUpFileServer(self, port, path):
```

Comment:
I'm not sure whether we should copy this code into a test. Maybe you can create a separate class in `test_torrent_def.py` that inherits from `TestAsServer` and put this test in there?